### PR TITLE
Zkpow improvements

### PR DESCRIPTION
**Pull request by 0xRomain**

### CODE REVIEW

File: group-generators/helpers/data-providers/talentlayer/index.ts [hunk 1]
```diff
@@ -217,4 +209,95 @@ export class TalentLayerProvider extends GraphQLProvider {
...
+
+    response.services.forEach((service) => {
+      if (
+        service.transaction &&
+        service.transaction.token.symbol == tokenSymbol
+      ) {
+        service.transaction?.payments.forEach((payment) => {
+          if (service.seller.address in users) {
+            users[service.seller.address] =
+              users[service.seller.address] + Number(payment.amount);
```
Comment (0xRomain) [inline group-generators/helpers/data-providers/talentlayer/index.ts:257]: use BigNumber

File: group-generators/generators/talentlayer/index.ts [hunk 2]
```diff
@@ -146,6 +144,35 @@ const generateDidUserMinimalEarnedOfTokenGroup = async (
...
+
+  const leaderboard = await talentLayerProvider.getTalentOfTheMonth(
+    topic,
+    period,
+    tokenSymbol,
+    leaderboardSize
+  );
+
+  return {
+    name: "talentlayer-talent-of-the-month",
```
Comment (0xRomain) [inline group-generators/generators/talentlayer/index.ts:164]: use variable to make a unique name, desc... 
Do it also for others

File: group-generators/helpers/data-providers/talentlayer/index.ts [hunk 3]
```diff
@@ -217,4 +209,95 @@ export class TalentLayerProvider extends GraphQLProvider {
...
   }
+
+  /**
+   * Get Talent of the month for a specific topic'
+   * @param topic
+   * @param period
+   * @param leaderboardSize
+   */
+  public async processGetTalentOfTheMonth(
+    topic: string,
```
Comment (0xRomain) [inline group-generators/helpers/data-providers/talentlayer/index.ts:220]: use new param object system

**Comment by MartinGbz** [comment]
Hey guys, please ping me when I can review and merge, thanks :)

**Pull request merged**